__tests__/endpoints.test.js

```diff
@@ -0,0 +1,28 @@
+const db = require("../db/connection");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const request = require("supertest");
+const topics = require("../db/data/test-data/topics");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
```

Comment:
This test is "brittle". If, for example, we were to add another property to the topics at a later date it would fail. Rather than defining `expectedTopics` and comparing the returned array you should iterate through the array and use a Jest matcher like `toMatchObject` and assert the properties of each topic and their data types (but not their actual values) for a test that is more robust. You should also assert the length of the array in the same test.